ironfish/src/wallet/wallet.ts

```diff
@@ -852,22 +852,6 @@ export class Wallet {
     return account.getBalance(assetId, confirmations)
   }
 
-  private async *getUnspentNotes(
```

Comment:
How will people know how to get unspent notes now in our SDK since this is going away? Is there a reason we're deleting this? It seems like this will make our SDK more confusing.

Reply:
it is a private function, so i'm assuming it won't be exported from our sdk. so i don't think they would be using it? or are you saying that they would use this as an example?